docs/cody/use-cases/vsc-tutorial.mdx

```diff
@@ -0,0 +1,126 @@
+# Cody Tutorial
+
+## Code Autocomplete
+
+![](https://storage.googleapis.com/sourcegraph-assets/blog/vs-code-onboarding-walkthrough-dec-2023-cody-autocomplete-tsx.gif)
+
+Start writing code and Cody will complete the line (or the entire function) for you. Hit tab to accept the suggestion.
```

Comment:
```suggestion
Start writing code and Cody will complete the line (or the entire function) for you. Click **Tab** to accept the suggestion.
```

Reply:
Tab isn't a button - it's the "Tab" key on the keyboard. I'll change to "Press **Tab**"